btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -81,6 +81,7 @@
           <IndividualPersonControlPercentage
             id="percentageOfShares"
             v-model="significantIndividual.percentOfShares"
+            type="number"
```

Comment:
I think this will break the current validation rules, can you verify?

FYI the api will accept strings that it can cast to numbers

Reply:
actually I tested it in the temp url and it seems to work, but the cypress test around it might need an update?